lib/reek/report/formatter.rb

```diff
@@ -42,6 +43,11 @@ def format_hash(warning)
         warning.yaml_hash
       end
 
+      # :reek:UtilityFunction
+      def format_code_climate_hash(warning)
+        CodeClimateFormatter.new(warning).to_hash
+      end
```

Comment:
Hm, rather than adding this to `SimpleWarningFormatter`, how about making `CodeClimateReport` use the `CodeClimateFormatter`?